Server/Utilities/sendSms.js

```diff
@@ -0,0 +1,20 @@
+import Nexmo from 'nexmo';
+
+const API_KEY = 'fa43188a';
+const API_SECRET = '6eec78df11a6d609';
+const nexmo = new Nexmo({
```

Comment:
I see you used environment variables in some places. Consider putting all keys, secrets, and app settings in your .env file.